modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -473,7 +469,8 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
          out.configs, ensemble.samples, inputs, Viz.output,
          file = file.path(settings$outdir,"SDA", "sda.output.Rdata"))
     #writing down the image - either you asked for it or nor :)
-    if (t%%2==0 | t==nt)  post.analysis.multisite.ggplot(settings, t, obs.times, obs.mean, obs.cov, FORECAST, ANALYSIS ,plot.title=control$plot.title, facetg=control$facet.plots, readsFF=readsFF)
+    if (control$plot == T & (t%%2==0 | t==nt)) post.analysis.multisite.ggplot(settings, t, obs.times, obs.mean, obs.cov, FORECAST, ANALYSIS ,plot.title=control$plot.title, facetg=control$facet.plots, readsFF=readsFF)
```

Comment:
Again, please write out `TRUE` and `FALSE`.

Also, again, always use `||` and `&&` in `if` statements, or wrap them in `any/all`.

Also, please use appropriate code style here: The `post.analysis...` block should be on its own line, surrounded by curly braces, and there should be spaces around all infix operators (i.e. `plot.title = control$plot.title`).

Reply:
Thanks Alexey. I'll try fixing as many of those as I edit previous code. Most of these are functions I didn't write so I'll try and make sure any new ones I happen to make follow more appropriate code styles :)